Makefile

```diff
@@ -15,4 +25,23 @@ update-charm-libs:
 check-dashboard-updates:
 	./scripts/check-dashboard-updates.sh
 
-.PHONY: help update-charm-libs check-dashboard-updates
+clean:
+	@echo "clean files"
+	@git clean -ffXd -e '!.idea' -e '!.vscode'
+	@echo "Cleaning existing build"
+	@rm -rf ${PROJECTPATH}/${CHARM_NAME}*.charm
+	@charmcraft clean
+	@echo "Remove download snap"
+	@rm -rf ${PROJECTPATH}/*.snap
+
+build: clean
+	@echo "Building charm"
+	@charmcraft -v pack
+
+download-snap:
+	wget -q https://github.com/canonical/openstack-exporter-operator/releases/download/rev2/golang-openstack-exporter_amd64.snap -O ./golang-openstack-exporter_amd64.snap
```

Comment:
Can we use snap directly here, e.g. `snap download golang-openstack-exporter --channel stable`.

Reply:
Ack. Maybe we can add some note or even better raise an issue, that we should use upstream if it will be released.

Reply:
@jneo8 Can you open an issue for this. Thanks